README-minishift.md

```diff
@@ -59,18 +59,18 @@ The steps in this walkthrough were tested with:
 ~~~sh
 minishift version
 
-minishift v1.25.0+90fb23e
+minishift v1.28.0+48e89ed
```

Comment:
This version of minishift already deploys `v3.11.0` by default, so maybe we want to remove the `--openshift-version` flags in the `minishift start` commands below?